### ISSUE TITLE
`carti bundle` accepting remote URIs instead of only local files

### ISSUE THREAD

**Issue by miltonjonat** [createdreamtech/carti#16]
Right now the `carti bundle` command only accepts a `<path>` parameter that points to a local file. However, it is expected that often an asset may already be published and available at a remote location, such as GitHub. As it is, using that asset would require the user to download the content, run `bundle`, and then run `publish` to "re-publish" the asset to the original URL.

To avoid all those steps, it would be great if the user could call `carti bundle` directly referring to the asset's remote location. For example:
```
carti bundle --type flash --name scrypt-hash --version 1.0.0 https://github.com/cartesi/descartes-tutorials/blob/master/dogecoin-hash/cartesi-machine/scrypt-hash.ext2?raw=true
```

**Comment by zcstarr** [comment]
This mildly tricky the choice here is about the uri being always compatible with http/https behavior where the assumption is that the request comes back with filename metadata in content headers, which may or may not be applicable if say the uri were dat://asdfasdf or ifps://asdfajwejka .

If bundle data can live anywhere, then my proposal will add an additional option called filename which will always be chosen if the uri fileName is not resolvable. 
 
 I'm just leaving this comment here for public record, if this comes back to be a, why's this here question for future generations :D 
 
 tl;dr I'll make this happen but will need to add a --fileName optional flag as an out for different storage providers if needed.
